test_sets/External_UCS-related_params.txt

```diff
@@ -0,0 +1,22 @@
+./test_scripts/Polices/External_UCS/009_ATF_P_TC_SnapshotPT_without_external_consent_status_groups_struct.lua Check_that_SDL_omits_external_consent_status_groups_struct_in_PTS_in_case_it_is_omitted_in_LPT
+./test_scripts/Polices/External_UCS/010_ATF_P_TC_SnapshotPT_with_external_consent_status_groups_struct.lua Check_that_SDL_adds_external_consent_status_groups_struct_to_PTS_in_case_it_is_defined_in_LPT
+./test_scripts/Polices/External_UCS/011_ATF_P_TC_SnapshotPT_without_disallowed_by_external_consent_entities_off_struct.lua Check_that_SDL_omits_disallowed_by_external_consent_entities_off_struct_in_PTS_in_case_it_is_omitted_in_LPT
+./test_scripts/Polices/External_UCS/012_ATF_P_TC_SnapshotPT_with_disallowed_by_external_consent_entities_off_struct.lua Check_that_SDL_adds_disallowed_by_external_consent_entities_off_struct_to_PTS_in_case_it_is_defined_in_LPT
+./test_scripts/Polices/External_UCS/013_ATF_P_TC_SnapshotPT_without_disallowed_by_external_consent_entities_on_struct.lua Check_that_SDL_omits_disallowed_by_external_consent_entities_on_struct_in_PTS_in_case_it_is_omitted_in_LPT
+./test_scripts/Polices/External_UCS/014_ATF_P_TC_SnapshotPT_with_disallowed_by_external_consent_entities_on_struct.lua Check_that_SDL_adds_disallowed_by_external_consent_entities_on_struct_to_PTS_in_case_it_is_defined_in_LPT
+./test_scripts/Polices/External_UCS/015_ATF_P_TC_PTU_without_external_consent_status_groups_struct.lua Check_that_SDL_processed_update_successfully_in_case_external_consent_status_groups_struct_is_omitted_in_PTU
+./test_scripts/Polices/External_UCS/016_ATF_N_TC_PTU_with_external_consent_status_groups_struct.lua Check_that_SDL_invalidate_update_in_case_external_consent_status_groups_struct_is_defined_in_PTU
+./test_scripts/Polices/External_UCS/017_ATF_P_TC_PTU_without_disallowed_by_external_consent_entities_off_struct.lua Check_that_SDL_processed_update_successfully_in_case_disallowed_by_external_consent_entities_off_struct_is_omitted_in_PTU
+./test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua Check_that_SDL_processed_update_successfully_in_case_disallowed_by_external_consent_entities_off_struct_is_defined_in_PTU
+./test_scripts/Polices/External_UCS/019_ATF_P_TC_PTU_without_disallowed_by_external_consent_entities_on_struct.lua Check_that_SDL_processed_update_successfully_in_case_disallowed_by_external_consent_entities_on_struct_is_omitted_in_PTU
+./test_scripts/Polices/External_UCS/020_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_on_struct.lua Check_that_SDL_processed_update_successfully_in_case_disallowed_by_external_consent_entities_on_struct_is_defined_in_PTU
+./test_scripts/Polices/External_UCS/021_ATF_N_TC_PTU_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua Check_that_SDL_invalidate_update_in_case_disallowed_by_external_consent_entities_off_struct_has_invalid_type_of_params_in_PTU
```

Comment:
@istoimenova, Let me clarify:
Since there is no dependency between parameters they can be verified separately
In this way there are following cases (c - value, desc - description):

{ v = "1", desc = "String" },
{ v = 1.23, desc = "Float" },
{ v = {}, desc = "Empty table" },
{ v = { entityType = 1, entityID = 1 }, desc = "Non-empty table" },
{ v = -1, desc = "Below Min" },
{ v = 129, desc = "Beyond Max" },
{ v = "", desc = "Blank" },
{ v = nil, desc = "Null" }

So 8 cases for each of 2 parameters. In total we'll have 16 cases